frontend/src/components/CardBack.tsx

```diff
@@ -29,16 +29,21 @@ interface MeaningContentProps {
   textColor: string;
   alternativeDepiction: string[];
   item: CardData;
+  headerColor?: string;
+  accentColor?: string;
```

Comment:
Why are we making these props? Can't we just use the use theme hook internally? Props are normally for consumers of the component to configure, I'm not sure the color is one of those. Or is this component so small that you don't what to call the useTheme there?